packages/sdk-ui-ts/src/utils/mappings.ts

```diff
@@ -310,5 +310,7 @@ export const validatorAddressToPathMap = {
     injvalcons1m8shfyz7cu7w0umgwgmhk59nmatzlvu7g56t6a:
     'injvaloper175x4lvxpwmmwt5m76yqz0ace8dwkjyjjj7sxdj.webp',
     injvaloper175x4lvxpwmmwt5m76yqz0ace8dwkjyjjj7sxdj:
-    'injvaloper175x4lvxpwmmwt5m76yqz0ace8dwkjyjjj7sxdj.webp'
```

Comment:
Why is this removed?

Reply:
I did not delete the line, this is the ATOM code editor so it pulled up and displays the change